newIDE/app/src/MainFrame/index.js

```diff
@@ -1791,8 +1844,10 @@ class MainFrame extends React.Component<Props, State> {
             return (
               <ClosableTab
                 label={editorTab.name}
+                type={editorTab.type}
                 key={editorTab.key}
                 active={isCurrentTab}
+                onSetPreview={() => this._setLayoutForPreview(editorTab.name)}
```

Comment:
Always need adjust here for get right name without (Events) on EventsSheet.

Reply:
name in EditorTabsHandler.js should be labelName, and layoutName for get real name of the layout without (Event) or (Extension) in the name :)

Reply:
I've not yet reviewed again but you're totally right that "name" is too broad, it should be called "label" (which is explicit that it's something shown on screen with a potential suffix).